src/app/dynamic-forms/fields/cinchy-child-form.directive.ts

```diff
@@ -141,6 +141,7 @@ export class ChildFormDirective {
       this.cinchyID = 0;
     } else {
       this.cinchyID = obj['Cinchy ID'];
+      if(this.cinchyID) this._ChildFormData.childFormData.rowId = this.cinchyID;
```

Comment:
Is there a more appropriate place to assign the _ChildFormData,childFormData.rowId ? Feels messy that the data passed into this component already has the cinchyId available to set as the rowId, but the child component here is the one setting value